.github/CODEOWNERS

```diff
@@ -17,6 +17,7 @@
 
 # content owners
 content/en/docs/instrumentation/cpp/     @open-telemetry/docs-approvers @open-telemetry/cpp-maintainers
+content/en/docs/instrumentation/net/     @open-telemetry/docs-approvers @open-telemetry/dotnet-approvers @open-telemetry/dotnet-instrumentation-approvers
```

Comment:
@austinlparker @reyang: are you ok with the addition of these teams, or would you prefer that I instead add the `-maintainers` teams? 

If you agree with the use of these teams, I'll address the access errors reported by GitHub.

Reply:
I'm okay +1 👍

Reply:
@austinlparker +1'd out-of-band